src/modules/app.js

```diff
@@ -0,0 +1,60 @@
+import ListTemplate from './template.js';
+import localGet from './localstorage.js';
+
+const todoContent = document.getElementById('listItem');
+const todoTask = document.getElementById('input');
+const displayList = () => {
+  todoContent.replaceChildren();
+  localGet().forEach((item, id) => {
+    todoContent.innerHTML += `
+        <div class="toDoItem">
+        <input class='item' id='check-${id}', "completed")' type='checkbox' ${item.Checked ? 'true' : 'false'} onclick='updateList(${id}, "completed")'>
```

Comment:
Good job! but it has a small issue, when I click on the checkbox it does complete the task, but when I refresh the page the checkbox does not remain checked. Please make sure to `check` the checkbox for completed tasks.